src/renderer/scss/component/_tooltip.scss

```diff
@@ -13,43 +13,57 @@
   }
 }
 
-.tooltip--on-formfield {
-  padding: 0;
-}
-
 .tooltip--icon {
   margin-top: 5px;
 }
 
 /* Tooltip text */
-.tooltip .tooltip__body {
-  background-color: var(--tooltip-bg);
-  font-family: 'metropolis-medium';
-  font-size: 12px;
-  color: var(--tooltip-color);
-  border-radius: 8px;
-  position: absolute;
-  z-index: 1;
-  width: 200px;
-  text-align: center;
-  white-space: pre-wrap;
-  padding: $spacing-vertical * 1/3;
-  visibility: hidden;
-}
+.tooltip {
+  .tooltip__body {
+    background-color: var(--tooltip-bg);
+    font-family: 'metropolis-medium';
+    font-size: 12px;
+    color: var(--tooltip-color);
+    border-radius: 8px;
+    position: absolute;
+    z-index: 1;
+    width: 200px;
+    text-align: center;
+    white-space: pre-wrap;
+    padding: $spacing-vertical * 1/3;
+    visibility: hidden;
+  }
+
+  .tooltip__body--short {
+    width: 130px;
+  }
+
+  .tooltip__body::after {
+    content: ' ';
+    width: 0;
+    height: 0;
+    position: absolute;
+    border-width: 5px;
+    border-style: solid;
+  }
 
-.tooltip .tooltip__body::after {
-  content: ' ';
-  width: 0;
-  height: 0;
-  position: absolute;
-  border-width: 5px;
-  border-style: solid;
+  &.tooltip--on-component {
+    .tooltip__body {
+      margin-top: 10px;
+    }
+  }
 }
 
-.tooltip--top .tooltip__body {
-  bottom: 100%;
-  left: 50%;
-  margin-left: -100px;
+.tooltip--top {
+  .tooltip__body {
+    bottom: 100%;
+    left: 50%;
```

Comment:
The margins on `<main>` appear to make this safe for realistic use cases, but what would happen if a tooltip is on a button with a width that  would extend past the canvas size?

Reply:
Tooltips can extend past the margin with no issues